### ISSUE TITLE
Question about weight decay

### ISSUE THREAD

**Issue by cnexah** [garrickbrazil/M3D-RPN#30]
Hello! Could I ask a question about the hyper-parameter weight decay?
In the paper, Sec 3.5, the weight decay is 0.9.
But in the code, the weight decay is set to be 0.0005.
So which one do you actually use during training?

Thank you very much!
